src/migrate.rs

```diff
@@ -0,0 +1,110 @@
+use {
+    crate::{
+        model::helpers::{get_project_by_project_id, upsert_project, upsert_subscriber},
+        Result,
+    },
+    mongodb::bson::doc,
+    serde::{Deserialize, Serialize},
+    std::collections::HashSet,
+};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ProjectData {
+    #[serde(rename = "_id")]
+    pub id: String,
+    pub identity_keypair: Keypair,
+    pub signing_keypair: Keypair,
+    pub app_domain: String,
+    pub topic: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Keypair {
+    pub private_key: String,
+    pub public_key: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ClientData {
+    #[serde(rename = "_id")]
+    pub id: String,
+    pub relay_url: String,
+    pub sym_key: String,
+    pub expiry: u64,
+    pub scope: HashSet<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct LookupEntry {
+    #[serde(rename = "_id")]
+    pub topic: String,
+    pub project_id: String,
+    pub account: String,
+    pub expiry: u64,
+}
+
+pub async fn migrate(mongo: &mongodb::Database, postgres: &sqlx::PgPool) -> Result<()> {
```

Comment:
Since this is called on each start, are we sure it's ok to call it again when the migration has already been done?